### ISSUE TITLE
Default template should not render collapsed children

### ISSUE THREAD

**Issue by jtrussell** [iVantage/angular-ivh-treeview#190]
I.e. the second suggestion [here](https://github.com/iVantage/angular-ivh-treeview#optimizations-and-known-limitations) should just be the default.

**Comment by Kuroky360** [comment]
Is  Angular's bind-once syntx  the best workaround for the performance now?

**Comment by jtrussell** [comment]
Using `ng-if` to not render collapsed children can give a huge performance boost - `bind-once` would be interesting to experiment with.

**Comment by Kuroky360** [comment]
Ok,so the next work is using ng-if to not render collapsed children for the v2.

**Comment by jtrussell** [comment]
Yep, the bulk of the work will be in updating the tests to not rely on a particular template - many tests will break with this change :(.

**Comment by Kuroky360** [comment]
So let's do it , and when to begin?

**Comment by jtrussell** [comment]
Feel free to dig in if you like :). You can submit PR's against the v2 branch.

**Comment by Kuroky360** [comment]
Yes,I will ~

**Comment by fhurta** [comment]
@Kuroky360 any progress with the performance boost? I'm hitting the limit too and the improvement would be very appreciated

**Comment by kuroky360** [comment]
@fhurta  sorry for that
